authz/src/main/java/io/grpc/authz/AuthorizationPolicyTranslator.java

```diff
@@ -167,25 +165,29 @@ private static Permission parseRequest(Map<String, ?> request) throws IllegalArg
   * If the policy cannot be parsed or is invalid, an exception will be thrown.
   */
   public static List<RBAC> translate(String authorizationPolicy) 
-  throws IllegalArgumentException, IOException {
+          throws IllegalArgumentException, IOException {
+    Object jsonObject = JsonParser.parse(authorizationPolicy);
+    if (!(jsonObject instanceof Map<?, ?>)) {
+      throw new IllegalArgumentException("failed to cast authorization policy");
```

Comment:
`"Authorization policy should be a JSON object. Found: " + (jsonObject == null ? null : jsonObject.getClass())`

Reply:
Done